warehouse/mssql/mssql.go

```diff
@@ -299,43 +299,55 @@ func (ms *HandleT) loadTable(tableName string, tableSchemaInUpload warehouseutil
 					recordInterface = append(recordInterface, value)
 				}
 			}
-			var recordInterface1 []interface{}
+			var finalColumnValues []interface{}
 			for index, value := range recordInterface {
 				valueType := tableSchemaInUpload[sortedColumnKeys[index]]
+				if value == nil {
+					pkgLogger.Errorf("MS : Found nil value for type : %s, column : %s", valueType, sortedColumnKeys[index])
+					finalColumnValues = append(finalColumnValues, nil)
+					continue
```

Comment:
Since you added a continue here, do you really need to attempt a conversion in the switch below again?

Reply:
The above check is just for nil values.
The conversion below is needed to convert to other types(int, date, boolean) from string.